docs/components/movement-sensor/_index.md

```diff
@@ -0,0 +1,672 @@
+
+**Returns:**
+
+- [(r3.Vector)](https://pkg.go.dev/github.com/golang/geo/r3#Vector): A 3D vector containing three floats representing the linear velocity in the x, y and z directions in millimeters per second.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK docs](https://pkg.go.dev/go.viam.com/rdk/components/movementsensor#MovementSensor).
+
+**Example usage:**
+
+```go
+mySensor, err := movementsensor.FromRobot(robot, "sensor1")
+
+// Get the current linear velocity of the movement sensor.
+linVel, _ := mySensor.LinearVelocity(context.TODO(), nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetAngularVelocity
+
+Report the current angular velocity about the x, y and z axes (as a 3D vector) in radians per second.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- [(Vector3)](https://python.viam.dev/autoapi/viam/components/movement_sensor/index.html#viam.components.movement_sensor.Vector3): A 3D vector containing three floats representing the angular velocity about the x, y and z axes in radians per second.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/movement_sensor/index.html#viam.components.movement_sensor.MovementSensor.get_angular_velocity).
+
+**Example usage:**
+
+```python
+my_sensor = MovementSensor.from_robot(robot=robot, name='my_movement_sensor')
+
+# Get the current angular velocity of the movement sensor.
+ang_vel = await my_sensor.get_angular_velocity()
+# Get the y component of angular velocity
+y_ang_vel = ang_vel.y
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map[string]interface{})](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [(r3.Vector)](https://pkg.go.dev/github.com/golang/geo/r3#Vector): A 3D vector containing three floats representing the angular velocity about the x, y and z axes in radians per second.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK docs](https://pkg.go.dev/go.viam.com/rdk/components/movementsensor#MovementSensor).
+
+**Example usage:**
+
+```go
+mySensor, err := movementsensor.FromRobot(robot, "sensor1")
+
+// Get the current angular velocity of the movement sensor.
+angVel, _ := mySensor.AngularVelocity(context.TODO(), nil)
+// Get the y component of angular velocity
+yAngVel := angVel.Y
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetLinearAcceleration
+
+Report the current linear acceleration in the x, y and z directions (as a 3D vector) in millimeters per second per second.

```

Comment:
is per second per second a useful way to say this? rather than per second squared or per second in each direction? If I even understand that correctly :)

Reply:
Yeah this is an accurate and meaningful way to say it. Per second squared is equivalent. Per second in each direction would be incorrect.